core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -248,9 +248,12 @@ private[spark] class SecurityManager(sparkConf: SparkConf)
   // configuration at a specified namespace. The namespace *must* start with spark.ssl.
   val fileServerSSLOptions = SSLOptions.parse(sparkConf, "spark.ssl.fs", Some(defaultSSLOptions))
   val akkaSSLOptions = SSLOptions.parse(sparkConf, "spark.ssl.akka", Some(defaultSSLOptions))
+  val dispatcherSSLOptions =
+    SSLOptions.parse(sparkConf, "spark.ssl.dispatcher", Some(defaultSSLOptions))
```

Comment:
It is used in core/src/main/scala/org/apache/spark/deploy/rest/RestSubmissionServer.scala which is not Mesos specific.